src/main/java/org/candlepin/subscriptions/tally/job/CaptureSnapshotsTaskManager.java

```diff
@@ -199,23 +230,23 @@ protected OffsetDateTime adjustTimeForLatency(
   }
 
   /**
-   * A class that is used to queue up account numbers as they are streamed from the DB so that they
-   * can be sent for updates in the configured batches.
+   * A class that is used to queue up org as they are streamed from the DB so that they can be sent
+   * for updates in the configured batches.
    */
-  private class AccountUpdateQueue {
+  private class OrgUpdateQueue {
```

Comment:
I'd rather go ahead and remove the update queue here since we don't use it in practice. This would mean building the message and then sending it immediately, without storing in an intermediate queue.

Reply:
Done